main.go

```diff
@@ -100,6 +101,12 @@ func main() {
 			Usage:  "Private key used in client TLS auth",
 			Value:  "",
 		},
+		cli.StringFlag{
```

Comment:
This is not needed.  If the env var is set, then it will be retrieved in the `installDockerGeneric`.  The reason we use `Before` to set the `MACHINE_STORAGE_PATH` is for testing and how the tests detect.

We shouldn't need this extra flag at all.